proposals/75:upgrade-16/vatDetails.js

```diff
@@ -0,0 +1,100 @@
+import dbOpenAmbient from 'better-sqlite3';
```

Comment:
I wonder why this file is here. I can't find anything that imports from it.

Reply:
It's used in the vaults and auctions upgrade. I dunno whether any parts were used in orchestration upgrades.